app/api/gradeSimulation/historyGraduationId/view/func.js

```diff
@@ -0,0 +1,23 @@
+const _ = require('lodash')
+const app = require('@/app')
+
+module.exports = async function (context) {
+  const GradeSimulation = app.model.gradeSimulations;
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555384)
```suggestion
  const GradeSimulation = app.model.gradeSimulations
```